### ISSUE TITLE
Create "What is Jupyter" page(s) + resources

### ISSUE THREAD

**Issue by ericsnekbytes** [jupyter/docs-team-compass#8]
We should create "What is Jupyter" page(s) + resources for subproject/meta documentation, as confusion about the Jupyter ecosystem is commonly observed by those working in the ecosystem:

> one of the repeated pieces of feedback we've heard from groups deploying Jupyter components (JupyterLab, JupyterHub, etc.) who aren't active in the community is that it's challenging to get a unified view of "what is Jupyter" and how the components work together.

This effort can include making a set of pages + some reusable diagrams such as the Jupyter client-server architecture, etc.

- PR: https://github.com/jupyter/jupyter/pull/728#discussion_r1613573049

**Comment by chbrandt** [comment]
A current view of "Jupyter", relating the projects. As proposed in issue #16 , a diagram can feature in the page.

**Comment by manics** [comment]
I think it'd be helpful if this also explained some of the confusing terminology and names. For example, "notebook" can refer to both the Jupyter Notebook server, and the notebook format. JupyterHub is a server, that runs instances of jupyter-server.

**Comment by ericsnekbytes** [comment]
@manics You're absolutely right. The problems you're describing come up frequently in conversations with the community, as many people report experiencing these same points of confusion. These are all key points that should be thoroughly explained and clarified in a "What is Jupyter" document.

**Comment by chbrandt** [comment]
Good point, @manics , thanks! I started addressing this issue in PR #23 . You are more than welcome to check if that's what you have in mind and suggest modifications if you have some time available.

**Comment by ericsnekbytes** [comment]
Can I steal some of this? I'm gonna open a PR to the jupyter/jupyter repo and we can collaborate more there perhaps?

**Comment by chbrandt** [comment]
Yes, sure
